.travis.yml

```diff
@@ -34,9 +34,10 @@ install:
   - $PYTHON_EXE -m easy_install --version
   - $PYTHON_EXE -m pip --version
   - $PYTHON_EXE -m tox --version
-  - $PYTHON_EXE -m pip install -U setuptools
```

Comment:
We need validate that works with current travis setuptools version and with newest setuptools version.
Then, we need execute the tests without a `pip -U` (old version) and run the tests again after a `pip -U` (new version)
```bash
# test with old setuptools version:
python -m tox -e coverage-erase,$TOXENV

# Update newest setuptools version:
$PYTHON_EXE -m pip install -U setuptools

# test with newest setuptools version:
python -m tox -e coverage-erase,$TOXENV
```